Subject: remove crosscompile check
From: Erwin Rol <erwin@erwinrol.com>

When cross compiling it refuses to use assembly since it assumes it can
not use the gen_matypes tool. But we fixed the gen_matypes so now the
test is not needed anymore.

Signed-off-by: Erwin Rol <erwin@erwinrol.com>
Signed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>

---
v1: written by Erwin Rol
v2: ported to Mesa-7.5 by Marc Kleine-Budde

 configure.ac |   16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

Index: Mesa-7.5/configure.ac
===================================================================
--- Mesa-7.5.orig/configure.ac
+++ Mesa-7.5/configure.ac
@@ -269,14 +269,14 @@ GLAPI_ASM_SOURCES=""
 AC_MSG_CHECKING([whether to enable assembly])
 test "x$enable_asm" = xno && AC_MSG_RESULT([no])
 # disable if cross compiling on x86/x86_64 since we must run gen_matypes
-if test "x$enable_asm" = xyes && test "x$cross_compiling" = xyes; then
-    case "$host_cpu" in
-    i?86 | x86_64)
-        enable_asm=no
-        AC_MSG_RESULT([no, cross compiling])
-        ;;
-    esac
-fi
+#if test "x$enable_asm" = xyes && test "x$cross_compiling" = xyes; then
+#    case "$host_cpu" in
+#    i?86 | x86_64)
+#        enable_asm=no
+#        AC_MSG_RESULT([no, cross compiling])
+#        ;;
+#    esac
+#fi
 # check for supported arches
 if test "x$enable_asm" = xyes; then
     case "$host_cpu" in
